### PR TITLE
[READY]cool new ammo boxes for shotguns

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl: 
add: you can now tactically reload shotguns with ammo boxes
refactor: better shotgun ammo boxes
/:cl:
todo:

- [x] - map edit old shotgun ammo boxes and change with new ones
- [x] - delete old shotgun ammo boxes

!! NEW !! CHANGES !! it now uses a do after so its more balanced

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -48,23 +48,23 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
...
-		new /datum/stack_recipe("virology airlock assembly", /obj/structure/door_assembly/door_assembly_viro, 4, time = 50, one_per_turf = 1, on_floor = 1), \
-		new /datum/stack_recipe("maintenance airlock assembly", /obj/structure/door_assembly/door_assembly_mai, 4, time = 50, one_per_turf = 1, on_floor = 1), \
-		new /datum/stack_recipe("external airlock assembly", /obj/structure/door_assembly/door_assembly_ext, 4, time = 50, one_per_turf = 1, on_floor = 1), \
-		new /datum/stack_recipe("external maintenance airlock assembly", /obj/structure/door_assembly/door_assembly_extmai, 4, time = 50, one_per_turf = 1, on_floor = 1), \
-		new /datum/stack_recipe("airtight hatch assembly", /obj/structure/door_assembly/door_assembly_hatch, 4, time = 50, one_per_turf = 1, on_floor = 1), \
-		new /datum/stack_recipe("maintenance hatch assembly", /obj/structure/door_assembly/door_assembly_mhatch, 4, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("standard airlock assembly", /obj/structure/door_assembly, 4, time = 50, one_per_turf = TRUE, on_floor = TRUE), \
```
Comment (bobbahbrown) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: undocumented changes

File: config/maps.txt [hunk 2]
```diff
@@ -13,21 +13,23 @@ Format:
...
 endmap
 
 map deltastation
-	minplayers 50
+	disabled
 endmap
 
-map runtimestation
+map runtimestation	
+	default
```
Comment (bobbahbrown) [inline config/maps.txt]: undocumented changes part 2

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: setting 1 to true is fine without changelog

File: config/maps.txt (hunk 2 above)
Comment (MrDoomBringer) [reply config/maps.txt]: nah this is a fine change keep it

Comment (ExcessiveUseOfCobblestone) [reply config/maps.txt]: amen

**Comment by DaxDupont** [comment]
Massive Buff to shotgun usage tbh

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -2144,6 +2144,42 @@
...
+/obj/machinery/atmospherics/pipe/simple/supply/hidden,
+/obj/effect/turf_decal/tile/neutral{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/neutral,
+/obj/effect/turf_decal/tile/neutral{
+	dir = 4
+	},
+/obj/item/ammo_box/shotgun/rubbershot{
+	step_x = 0;
```
Comment (SpaceManiac) [inline _maps/map_files/BoxStation/BoxStation.dmm]: `step_x` is verboten, these vars should be removed

File: config/maps.txt [hunk 4]
```diff
@@ -13,21 +13,23 @@ Format:
 endmap
 
 map boxstation
```
Comment (SpaceManiac) [inline config/maps.txt]: changes to this file should be reverted

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Comment by MrDoomBringer** [comment]
I can make some nicer sprites but i'll do it in a separate PR (soonTM)

**Comment by anconfuzedrock** [comment]
poor bandolier

**Comment by subject217** [comment]
ya shotguns werent good enough lol they needed to be able to be reloaded instantly, their current greatest drawback

**Comment by BeeSting12** [comment]
Don't make shotguns instantly reloadable thats a bad idea.

**Comment by Borisvanmemes** [comment]
As much as i love shotguns this makes them EVEN BETTER, AND THEY'RE ALREADY SOME OF THE BEST SHIT THE STATION HAS.

**Comment by anconfuzedrock** [comment]
Honestly this wouldn’t even make sense at all unless its for bandoliers. Already imagining a lethal injections dart slug murderbone

**Comment by Potato-Masher** [comment]
I plan to abuse this to hell and back if merged.

**Comment by lolman360** [comment]
wtf i didn't know we were rebasing to goon

**Comment by ExcessiveUseOfCobblestone** [comment]
It's a sprite PR with a major balance change as a secondary,who would possibly cre- oh nevermind

**Comment by epochayur** [comment]
bad bad bad idea

**Comment by MrDoomBringer** [comment]
ah fuck he added a do_after
i was so hype for the new stupidly broken bullet hose shotguns

**Comment by Tlaltecuhtli** [comment]
reloading is somewhat similar to old reload, but no longer carpal tunnel inducing

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 5]
```diff
@@ -65,4 +65,94 @@
 /obj/item/ammo_box/foambox/riot
 	icon_state = "foambox_riot"
 	ammo_type = /obj/item/ammo_casing/caseless/foam_dart/riot
-	materials = list(MAT_METAL = 50000)
\ No newline at end of file
+	materials = list(MAT_METAL = 50000)
+
+/obj/item/ammo_box/shotgun
+	name = "ammo box (12g)"
+	icon_state = "12g_box"
```
Comment (MrDoomBringer) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm]:
```suggestion
	icon_state = "12g_box"
	desc = "A tactical ammunition box designed to easily reload shotguns.
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]
Could be a little more clear that it can be used to reload things easily

**Comment by optimumtact** [comment]
I'm like 75% convinced bandoliers already do this for shotguns and that's why shotguns don't normally do it to ammo boxes

**Comment by Tlaltecuhtli** [comment]
https://github.com/tgstation/tgstation/blob/a9849e7a54addf333bbd37cf5d584299d4a32945/code/game/objects/items/storage/belt.dm#L506 no

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 6]
```diff
@@ -65,4 +65,95 @@
...
+	desc = "A tactical ammunition box designed to easily reload shotguns."
+	ammo_type = /obj/item/ammo_casing/shotgun
+	max_ammo = 8
+	start_empty = TRUE
+
+/obj/item/ammo_box/shotgun/Initialize(mapload)
+	if(mapload)
+		start_empty = FALSE
+	..()
```
Comment (ShizCalev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm]:
```suggestion
	return ..()
```

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 7]
```diff
@@ -65,4 +65,95 @@
...
+
+/obj/item/ammo_box/shotgun/attack_self(mob/user)
+	..()
```
Comment (ShizCalev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm]:
```suggestion
	. = ..()
```

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — CHANGES_REQUESTED** [review]
dart_box sprite has a rogue pixel.

**Comment by optimumtact** [comment]
this is too powerful imo, shotguns are already the best weapon

**Pull request closed without merging**